packages/react-components/react-avatar/src/stories/AvatarGroup/AvatarGroupIndicator.stories.tsx

```diff
@@ -31,20 +26,30 @@ const names = [
 
 export const Indicator = () => {
   const styles = useStyles();
-  const countAvatarGroupItems = names.map(name => <AvatarGroupItem name={name} key={'spread-' + name} />);
-  const iconAvatarGroupItems = names.map(name => <AvatarGroupItem name={name} key={'stack-' + name} />);
-  const countPartitionedItems = getPartitionedAvatarGroupItems(countAvatarGroupItems);
-  const iconPartitionedItems = getPartitionedAvatarGroupItems(iconAvatarGroupItems);
+  const countPartitionedItems = partitionAvatarGroupItems({ items: names });
+  const iconPartitionedItems = partitionAvatarGroupItems({ items: names });
```

Comment:
(nit) you don't need to do the same thing twice. Both examples could share the same array.
```suggestion
  const partitionedItems = partitionAvatarGroupItems({ items: names });
```